<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="2" failures="2">
  <testsuite name="Root Suite" timestamp="2020-03-29T00:40:40" tests="0" file="cypress/integration/apps/projects/PR23.js" failures="0" time="0">
  </testsuite>
  <testsuite name="23 - I can work with samples in running experiments" timestamp="2020-03-29T00:40:40" tests="2" failures="2" time="0">
    <testcase name="23 - I can work with samples in running experiments &quot;before all&quot; hook for &quot;PR-23-1 Should not add samples to a running experiment&quot;" time="0" classname="&quot;before all&quot; hook for &quot;PR-23-1 Should not add samples to a running experiment&quot;">
      <failure message="cy.exec(&apos;esp  -P 8080 -H 127.0.0.1 -u admin@localhost -p admin seed --no-overwrite cypress/fixtures/apps/projects/PR23seed.yml&apos;) failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
INFO:root:2020-03-28 19:40:40,504:Creating Protocol: PR23 standard protocol
INFO:root:2020-03-28 19:40:41,187:Creating Protocol: PR23 standard protocol2
INFO:root:2020-03-28 19:40:41,333:Creating W...

Because this error occurred during a &apos;before all&apos; hook we are skipping the remaining tests in the current suite: &apos;23 - I can work with sample...&apos;" type="CypressError"><![CDATA[CypressError: cy.exec('esp  -P 8080 -H 127.0.0.1 -u admin@localhost -p admin seed --no-overwrite cypress/fixtures/apps/projects/PR23seed.yml') failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
INFO:root:2020-03-28 19:40:40,504:Creating Protocol: PR23 standard protocol
INFO:root:2020-03-28 19:40:41,187:Creating Protocol: PR23 standard protocol2
INFO:root:2020-03-28 19:40:41,333:Creating W...

Because this error occurred during a 'before all' hook we are skipping the remaining tests in the current suite: '23 - I can work with sample...'
    at cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138017:3)
    at http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:125593:26
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:163012:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
    <testcase name="23 - I can work with samples in running experiments &quot;after all&quot; hook for &quot;PR-23-1 Should not add samples to a running experiment&quot;" time="0" classname="&quot;after all&quot; hook for &quot;PR-23-1 Should not add samples to a running experiment&quot;">
      <failure message="cy.exec(&apos;esp  -P 8080 -H 127.0.0.1 -u admin@localhost -p admin seed -f -d cypress/fixtures/apps/projects/PR23seed.yml&apos;) failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
INFO:root:2020-03-28 19:40:46,434:Saving data in Experiment PR23 for Protocol: PR23 standard protocol.
Traceback (most recent call last):
  File &quot;/Users/cj/bitrepos/esp/develop/bin/.venv/bin/esp&quot;, ...

Because this error occurred during a &apos;after all&apos; hook we are skipping the remaining tests in the current suite: &apos;23 - I can work with sample...&apos;" type="CypressError"><![CDATA[CypressError: cy.exec('esp  -P 8080 -H 127.0.0.1 -u admin@localhost -p admin seed -f -d cypress/fixtures/apps/projects/PR23seed.yml') failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
INFO:root:2020-03-28 19:40:46,434:Saving data in Experiment PR23 for Protocol: PR23 standard protocol.
Traceback (most recent call last):
  File "/Users/cj/bitrepos/esp/develop/bin/.venv/bin/esp", ...

Because this error occurred during a 'after all' hook we are skipping the remaining tests in the current suite: '23 - I can work with sample...'
    at cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138017:3)
    at http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:125593:26
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:163012:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
  </testsuite>
</testsuites>